packages/api-tests/test/window.spec.js

```diff
@@ -54,16 +54,328 @@ describe('Window API', function(done) {
     return executeAsyncJavascript(app.client, script);
   });
 
+  it('Should close the window', function() {
```

Comment:
Think having all of these in a describe would be nice so we can group the window option tests together, the window method tests together etc